pubspec.yaml

```diff
@@ -1,31 +1,31 @@
 name: nyxx_commands
-version: 4.3.0
+version: 4.3.1
 description: A framework for easily creating slash commands and text commands for Discord using the nyxx library.
 
 homepage: https://github.com/nyxx-discord/nyxx_commands/blob/main/README.md
 repository: https://github.com/nyxx-discord/nyxx_commands
 issue_tracker: https://github.com/nyxx-discord/nyxx_commands/issues
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
Tiny nit but I'd rather keep the single quotes as that is the default

Reply:
for sure! I think my code editor may have changed this automatically dunno why